src/components/WalletModal/PendingView.tsx

```diff
@@ -67,41 +64,31 @@ const LoadingWrapper = styled.div`
 `
 
 export default function PendingView({
-  connector,
-  error = false,
-  setPendingError,
-  tryActivation,
+  walletOptionKey,
+  hasError = false,
+  renderHelperText = () => null,
+  onClickTryAgain,
 }: {
-  connector?: AbstractConnector
-  error?: boolean
-  setPendingError: (error: boolean) => void
-  tryActivation: (connector: AbstractConnector) => void
+  walletOptionKey?: string // key of SUPPORTED_WALLETS
+  hasError?: boolean
+  renderHelperText?: () => React.ReactNode
+  onClickTryAgain: () => any
```

Comment:
I don't see that func return anything. so `()=> void`

Reply:
Updated